### PR TITLE
fix: some rules not considering options correctly

### PR DESCRIPTION
**Pull request by rafaelss95**
The following rules aren't behaving correctly at the moment:

- max-inline-declarations;
- no-input-prefix;
- prefer-inline-decorator;

The problem with these rules are that it's only considering the options from the second element (in position 1) of the `array`, which is totally incorrect.

<hr>

Example for `max-inline-declarations`:

```JSON
{
  "rules": {
    "max-inline-declarations": [true, { "template": 2 }]
  }
}
```

```typescript
@Component({
  selector: 'app-root',
  template: `
    <div>
     something here
    </div>
  `,
  styleUrls: ['./app.component.css']
})
export class AppComponent {}
```

**Expected behavior:**
Failure.

**Actual behavior:**
No failure.

<hr>

This fixes the wrong behavior and respective tests, with some refactor.

**PS:** The spec file of `max-inline-declarations` was wrongly named.

### CODE REVIEW

**Comment by rafaelss95** [comment]
@wKoza can you review?

**Comment by wKoza** [comment]
@rafaelss95 , we have an another problem with the rule `max-inline-declarations` in your example. This rule compute all the lines between `' ... '`. Your use case, the rule count `5` lines but I think that `3` is the good value. So, we should exclude a first and the last line when it is empty lines.

**Comment by rafaelss95** [comment]
@Wkoza Hmm, yes, I noticed this problem now. So, should we exclude all empty lines from calculation or only the first and last?

File: src/maxInlineDeclarationsRule.ts [hunk 1]
```diff
@@ -59,8 +88,9 @@ export class MaxInlineDeclarationsValidator extends NgWalker {
   private validateInlineTemplate(metadata: ComponentMetadata): void {
     if (this.hasInlineTemplate(metadata) && this.getTemplateLinesCount(metadata) > this.templateLinesLimit) {
       const templateLinesCount = this.getTemplateLinesCount(metadata);
-      const msg = `Inline template lines limit exceeded. Defined limit: ${this.templateLinesLimit} / template lines: ${templateLinesCount}`;
```
[the remark this answers is not in the document]
Comment (rafaelss95) [reply src/maxInlineDeclarationsRule.ts]: No problem, I'll put it again.

**Comment by wKoza** [comment]
The main reason for this rule is to increase readabilty of our components. All the HTML source code obscure the component's implementation, so, I think we can compute comments and the other empty lines.

**Pull request merged**